doc/usage/extensions/autosummary.rst

```diff
@@ -201,6 +201,25 @@ also use these config values:
 
    .. versionadded:: 2.1
 
+   .. versionchanged:: 4.3
+
+      If ``autosummary_ignore___all__`` is ``False``, this configuration value
+      is ignored for members listed in ``__all__``.
+
+.. confval:: autosummary_ignore___all__
+
+   If ``False`` and a module has the ``__all__`` attribute set, autosummary
+   documents every member listed in ``__all__`` and no others. Default is
+   ``True``
+
+   Note that if an imported member is listed in ``__all__``, it will be
+   documented regardless of the value of ``autosummary_imported_members``. To
+   match the behaviour of ``from module import *``, set
+   ``autosummary_ignore___all__`` to False and ``autosummary_imported_members``
+   to True.
+
+   .. versionadded:: 4.3
```

Comment:
Unfortunately, we'll ship 4.3.0 soon. So it's timed out to merge this into it. Please change this to 4.4.